chemprop/cli/train.py

```diff
@@ -638,6 +641,63 @@ def build_splits(args, format_kwargs, featurization_kwargs):
     return train_data, val_data, test_data
 
 
+def summarize(args, dataset: _MolGraphDatasetMixin) -> tuple[list, list]:
+    if args.task_type == "spectral":
+        pass
```

Comment:
This will cause the code to error if task_type is spectral as `summarize` will return `None` and python can't unpack that into `column_headers, table_rows`. I think it would be better to have this `if args.task_type == "spectral"` check in `build_datasets` before running the for loop to summarize the datasets. That way if the task type is spectral it will just skip all the summarizing code entirely.

Reply:
just made this change.